README.md

```diff
@@ -94,7 +94,8 @@ Autofaiss allows users to build indices in Spark, you need to do the following s
 2. Prepare your embeddings files.
 3. Create a spark session before using `build_index` (optional), if you don't create it, a default session would
     be created with the least configuration.
-
+### Producing N indices
+In the distributed mode, you can generate an index larger than your current available memory by setting `nb_indices_to_keep`
```

Comment:
can you add here information about the ordering of the indices and their naming?
btw if it's not done already, could you zero-pad the names ? (for examples should be index01 index02 ... index11 and not index1 index2 .. index11) ; otherwise it often causes confusing due to alphabetical sorting being used

Reply:
also pointing to an usage exemple would be good (reading all indices and searching them sequentially / merging the results) since it's not completely trivial

Reply:
Good advice. Will do.